mentors/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from users.models import CustomUser
+default_user = CustomUser.objects.get(email='ecell@gmail.com')
```

Comment:
Don't do like this

Reply:
Sir this an outdated file , we are not using default user anymore. Instead we are setting it to null